src/Adapter/PHPCR/DoctrinePHPCRAdapter.php

```diff
@@ -18,8 +18,14 @@
 use Doctrine\Persistence\ManagerRegistry;
 use Sonata\Doctrine\Adapter\AdapterInterface;
 
+@trigger_error(
+    'The Sonata\Doctrine\Adapter\PHPCR\DoctrinePHPCRAdapter class is deprecated'
+    .' since sonata-project/doctrine-extensions 1.x, to be removed in version 2.0.',
+    \E_USER_DEPRECATED
+);
+
 /**
- * This is a port of the DoctrineORMAdminBundle / ModelManager class.
+ * NEXT_MAJOR: Remove this class.
```

Comment:
You should add an `@deprecated` comment so your IDE can highlight deprecation usage.